osCam/core/views.py

```diff
@@ -19,7 +19,7 @@ def __init__(self):
         self.avg = None
         self.out = None
         self.numframes = 0
-        self.record = True
+        self.record = False
```

Comment:
Good call committing a version with the default recording set to false

Reply:
unintentional, lucky change. I should be more careful overall but I agree this is a nice default.